superset-frontend/src/SqlLab/components/QueryTable.jsx

```diff
@@ -188,22 +188,22 @@ const QueryTable = props => {
         );
         q.actions = (
           <div>
-            <Link
-              className="fa fa-pencil m-r-3"
+            <IconTooltip
+              className="fa fa-pencil m-r-3 pointer"
```

Comment:
if `pointer` is a bootstrap class, for the cursor, it might be good to either
a) add a `pointer` prop to the `IconTooltip` component, and add the style conditionally there
b) just... add the cursor style to the IconTooltip component by default if this should _always_ be the case ¯\\\_(ツ)_/¯

Reply:
or... maybe the `role` attribute makes sense in this case, too?

Reply:
I'll add a property because we have icons that only display information but are not clickable.

Reply:
I was thinking... `IconTooltip` has a `className` property and a `style` property. So I shouldn't add a `cursor` property because that is part of a style. I could pass a property value of `style={{cursor: pointer}}` but since I'm using `className` I think that passing the pointer class is better because all the style of the component is being defined in a single place.

Reply:
Totally agree that the `className` is better than the `style`! 100%

What I was suggesting (but not at all requiring!) was adding a `pointer` prop to `IconTooltip`, i.e. `<IconTooltip pointer className=...>`

The IconTooltip would pass that prop into the `StyledSpan` you added, which would then use it to render the styles needed, like so:

```
const StyledSpan = styled.span`
  cursor: ${({ pointer }) => pointer ? 'pointer' : 'default'};
```

... but that may be overkill ;)